Example code doesn't compile (actix-web v2.0.0, rustc v.1.4.0)

I tried to compile the example from the readme. My cargo.toml looks like this:
```toml[package]
name = ...
version = "0.1.0"
authors = ...
edition = "2018"

[dependencies]
actix-web = "2.0.0"
```

The example I copied:
```rust
use actix_web::{get, web, App, HttpServer, Responder};

#[get("/{id}/{name}/index.html")]
async fn index(info: web::Path<(u32, String)>) -> impl Responder {
    format!("Hello {}! id:{}", info.1, info.0)
}

#[actix_rt::main]
async fn main() -> std::io::Result<()> {
    HttpServer::new(|| App::new().service(index))
        .bind("127.0.0.1:8080")?
        .run()
        .await
}
```

The error messages I get

```bash
error[E0433]: failed to resolve: use of undeclared type or module `actix_rt`
 --> src/main.rs:8:3
  |
8 | #[actix_rt::main]
  |   ^^^^^^^^ use of undeclared type or module `actix_rt`

error[E0277]: `main` has invalid return type `impl std::future::Future`
 --> src/main.rs:9:20
  |
9 | async fn main() -> std::io::Result<()> {
  |                    ^^^^^^^^^^^^^^^^^^^ `main` can only return types that implement `std::process::Termination`
  |
  = help: consider using `()`, or a `Result`

error: aborting due to 2 previous errors
```

If there's something I'm doing wrong, it should be more clear in the readme what is necessary to get the example compiling. I only find "Supported Rust version: 1.39 or later", so I don't think there's anything wrong here.

--- a/README.md
+++ b/README.md
@@ -41,6 +41,16 @@ Actix web is a simple, pragmatic and extremely fast web framework for Rust.
 
 ## Example
 
+Dependencies:
+
+```toml
+[dependencies]
+actix-web = "2"
+actix-rt = "1"
+```
+
+Code:
+
 ```rust
 use actix_web::{get, web, App, HttpServer, Responder};
 
@@ -65,7 +75,7 @@ async fn main() -> std::io::Result<()> {
 * [Multipart streams](https://github.com/actix/examples/tree/master/multipart/)
 * [Simple websocket](https://github.com/actix/examples/tree/master/websocket/)
 * [Tera](https://github.com/actix/examples/tree/master/template_tera/) /
-  [Askama](https://github.com/actix/examples/tree/master/template_askama/) templates
+* [Askama](https://github.com/actix/examples/tree/master/template_askama/) templates
 * [Diesel integration](https://github.com/actix/examples/tree/master/diesel/)
 * [r2d2](https://github.com/actix/examples/tree/master/r2d2/)
 * [OpenSSL](https://github.com/actix/examples/tree/master/openssl/)

